nucliadb/nucliadb/ingest/orm/entities.py

```diff
@@ -107,9 +107,12 @@ async def update_entities(self, group: str, entities: Dict[str, Entity]):
         intact. Use `delete_entities` to delete them instead.
 
         """
+        if not self.entities_group_exists(group):
+            raise KeyError(f"Entities group '{group}' doesn't exist")
```

Comment:
Can you raise a more explicit exception? Otherwise we could be covering up a hidden KeyError bug when handling this one